tests/test_issues/test_issue_2224.py

```diff
@@ -0,0 +1,52 @@
+import unittest
+from unittest import TestCase
+from linkml_runtime.utils.schemaview import SchemaView
+from jsonasobj2 import JsonObj
+
+from tests.test_issues.environment import env
+
+
+class Issue2224TestCase(TestCase):
```

Comment:
CI already runs pytest, and if the goal longterm is to move to pytest, might be a good idea to write things as pytest tests from here on out :)